packages/validator/src/cli-validator/run-validator.js

```diff
@@ -168,16 +168,6 @@ const processInput = async function(program) {
 
       const fileExtension = ext.getFileExtension(validFile);
       if (fileExtension === 'json') {
-        // find and fix trailing commas
-        const match = originalFile.match(/,\s*[}\]]/m);
-        if (match) {
-          const chars = originalFile.substring(0, match.index);
-          const lineNum = (chars.match(/\n/g) || []).length + 1;
-          const msg = `Trailing comma on line ${lineNum} of file ${validFile}.`;
-          printError(chalk, chalk.red(msg));
-          exitCode = 1;
-          originalFile = originalFile.replace(/,(\s*[}\]])/gm, '$1');
-        }
```

Comment:
I agree that this is a good idea but I think this would be a good opportunity to utilize the logging feature that is in the works. IIRC, we added this code because users were struggling with low-detail error messages from the JSON parser that were time consuming to debug.

This would be a good candidate for a "debug" level log that says something like "JSON may have a trailing command" (intentionally vague because of the false positive you described). Perhaps only if the `JSON.parse` fails. Just something to give the user more insight into what their issue may be.

This certainly doesn't need to happen in this PR, I just wanted to capture it in a comment. I can open an internal issue linking this commit reminding us to re-add it once the logger is implemented. Or this PR can leave the console.log and nothing else.

What do you think?